tests/regression/test_issue_1826.cpp

```diff
@@ -0,0 +1,40 @@
+#define CL_HPP_ENABLE_EXCEPTIONS
```

Comment:
The license/copyright header is missing.

Reply:
fixed